src/Globe/Globe.js

```diff
@@ -187,9 +186,11 @@ Globe.prototype.setLayerVisibility = function(id, visible) {
     this.layersConfiguration.setLayerVisibility(id, visible);
 
     var cO = function(object) {
```

Comment:
@mlamure can you please update your code to my new modif? i can not see your point and line any more.